init_pioneer.sh.cmakein

```diff
@@ -0,0 +1,45 @@
+#! /bin/bash
+
+PIONEER_DATA_DIR="@_PIONEER_DATA_DIR@"
+SOURCE_DIR=$APPDIR/usr/share/data
+BIN_DIRECTORY=$APPDIR/usr/bin
+
+if [ "$1" = editor ]
+then
+    shift
+    if [ ! -d "$PIONEER_DATA_DIR" ]; then
+        echo "Copying pioneer data directory from $SOURCE_DIR to $PIONEER_DATA_DIR"
+        cp -r "$SOURCE_DIR" "$PIONEER_DATA_DIR"
```

Comment:
Hi,

As discussed with @Web-eWorks , it is project requirement for the user to have the access to the data files for modding purposes and downloading the 

When the user runs the appimage for the first time it will copy the data outside of the appimage bundle, this is a one time operation.

Reply:
Appimage is very different from installation, it is bundled as a single binary which is mounted in /tmp location on execution, as the requirement is to have access to modify data files, it copies it if the PIONEER_DATA_DIR is missing, there is an option to refresh it with the files in the bundle if needed

Reply:
I know how AppImage works.

The users don't have access to Pioneer's installed data files on a regular Linux install, either. So the proper way to handle this would be for Pioneer to look into the ~/.local/share/pioneer folder first and then the regular data directory second.

Copying the whole directory of data files to a random directory is absolutely not a good design.

Reply:
> The users don't have access to Pioneer's installed data files on a regular Linux install, either.

On a regular linux install, users can still inspect the entire contents of the game data directory, typically from `/usr/share/pioneer`.

> it is project requirement for the user to have the access to the data files for modding purposes

@VasanthBalguri 
It's not a hard requirement to merge this PR for the data to be accessible, but I did mention it as a downside of the appimage deployment process - I apologize for the miscommunication.

Thank you for looking into ways to solve the problem of making the game data accessible; it seems it's a problem we'll simply have to live with. I would prefer for users to have the same modding experience regardless of deployment vector, but I don't think there's a sane way to achieve that.